tests/contrib/test_opentelemetry.py

```diff
@@ -143,6 +143,16 @@ def query(self) -> str:
     def signal(self) -> None:
         self._signal_count += 1
 
+    @workflow.update
+    def update(self) -> int:
+        self._signal_count += 1
+        return self._signal_count
+
+    @update.validator
+    def update_validator(self) -> None:
+        print("Actually in validator")
```

Comment:
```suggestion
        workflow.logger.info("Actually in validator")
```

To avoid print calls in code

Reply:
Just accidentally left over anyway. Removed.